packages/components/.storybook/config.tsx

```diff
@@ -54,24 +50,10 @@ addDecorator(withA11y);
 addDecorator(withKnobs);
 addDecorator(withGlobal);
 addDecorator(ThemeDecorator);
-addParameters({
-  viewport: {
-    viewports,
-  },
-});
+addParameters({ viewport: { viewports } });
 
 // Option defaults.
-addParameters({
-  options: {
-    theme: themes.dark,
-  },
-});
-
-// automatically import all files ending in *.stories.tsx
-const req = require.context('../src', true, /.stories.tsx$/);
-
-const loadStories = () => {
-  req.keys().forEach(req);
-};
+addParameters({ options: { theme: themes.dark } });
 
-configure(loadStories, module);
+// automatically import all files ending in *.stories.js
+configure(require.context('../src', true, /\.stories\.tsx$/), module);
```

Comment:
changed the declaration so that we can use their new format, see collapsible/stories.tsx